### ISSUE TITLE
console prints spam

### ISSUE THREAD

**Issue by garrotte13** [robot256/Vehicle-Wagon-2#14]
Hi!
![image](https://user-images.githubusercontent.com/95149904/218710788-08e7e1df-8cb7-4ac2-8b16-ab2131616687.png)
I suppose that's not good to print a message in console every time a winch is used. Console is for debugging, testing, scripts cheating and other abnormal activities monitoring, not for standard activities. Another way could be a flying-text entity use (I use one entity prototype for all mod messages) - works fine for such rare messages.
If my humble opinion is completely wrong at least a mod setting per player to turn those hints off could solve the problem.

P.S. What do you think about moving winch out of items into pure shortcuts?

**Comment by garrotte13** [comment]
Looks like there are some useful messages for typical user inaccuracy mistakes. But can be easily displayed via player.create_local_flying_text .

For example
use
```
player.create_local_flying_text({
				position = position,
				time_to_live = 250,
				text = {"vehicle-wagon2.train-in-motion-error"}
			})
```
instead of
`player.print{"vehicle-wagon2.train-in-motion-error"}`

**Comment by robot256** [comment]
This has been implemented for the 2.0 release
